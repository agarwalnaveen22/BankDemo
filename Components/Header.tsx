import React from 'react';
import {View, Text} from 'react-native';
import EStyleSheet from 'react-native-extended-stylesheet';

const Header = (props: {title: string}) => {
  return (
    <View style={styles.header}>
      <Text style={styles.headerText}>{props.title}</Text>
    </View>
  );
};

const styles = EStyleSheet.create({
  header: {
    width: '100%',
    height: '50rem',
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#6db2f7',
  },
  headerText: {
    fontSize: '25rem',
    color: '#ffffff',
  },
});

export default Header;
