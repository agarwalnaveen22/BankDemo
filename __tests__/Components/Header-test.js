import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../Components/Header';

describe('test header container component', () => {
  it('match snapshot', () => {
    const snap = renderer.create(
    <Header 
        text="Register"
    />
    ).toJSON();
    expect(snap).toMatchSnapshot();
  });
});