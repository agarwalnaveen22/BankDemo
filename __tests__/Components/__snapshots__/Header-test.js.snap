// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header container component match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#6db2f7",
      "height": 98.68421052631578,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 49.34210526315789,
      }
    }
  />
</View>
`;
